extensions/2.0/Khronos/KHR_texture_procedurals/README.md

```diff
@@ -0,0 +1,1351 @@
+
+## Contributors
+
+- Ashwin Bhat, Autodesk, ashwin.bhat@autodesk.com
+- Alexey Knyazev, Individual Contributor, [@lexaknyazev](https://github.com/lexaknyazev)
+- Bernard Kwok, Individual Contributor, [@kwokcb](https://github.com/kwokcb)  
+- Ed Mackey, AGI, [@emackey](https://twitter.com/emackey)
+- Lutz Kettner, NVIDIA, lkettner@nvidia.com
+- Richard Sahlin, IKEA [@rsahlin](https://github.com/rsahlin)
+- Timothy Blut, NVIDIA, tblut@nvidia.com
+
+Copyright 2024 The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension provides a standardized way to represent procedural graphs which can be mapped to material channels in glTF.
+
+### Table of Contents
+
+- [Motivation](#motivation)
+- [Definitions](#definitions)
+- [Extension Declaration](#extension-declaration)
+- [Representation](#representation)
+  - [Data Types](#data-types)
+  - [Procedural Graphs](#procedural-graphs)
+  - [Procedural Nodes](#procedural-nodes)
+  - [Node Graph Connections](#node-graph-connections)
+  - [Material Binding](#material-binding)
+  - [Resource Binding](#resource-binding)
+    - [Uniform Binding](#uniform-binding)
+    - [Texture Binding](#texture-binding)
+    - [Input Stream Binding](#input-stream-binding)
+  - [Procedural Definitions](#procedural-definitions)
+    - [Structure](#structure)
+- [Structure Review](#structure-review)
+- [JSON Schema](#json-schema)
+- [Checkerboard Example](#checkerboard-example)
+- [Resources](#resources)
+
+### Motivation
+
+Textures represented as procedural graphs provides a way to extend the capabilities of glTF materials beyond what is possible with traditional texture maps. Key objectives includes:
+
+1. **Interoperability**: Adhere to an industry a standard node schema (MaterialX) with a runtime friendly JSON representation. 
+
+    More specifically __only__ nodes defined in the MaterialX specification are supported. This ensures that the procedural graphs can be easily converted to MaterialX and USDShade graphs.
+
+    For the first version of this extension nodes which are used to define shading models are not allowed. Please refer to the [resources](#resources) section for links to supported  MaterialX node definitions.
+
+2. **Fidelity**: Provide the ability to generate complex patterns, noise, or other effects that currently must be "baked" into texture maps. Provide the ability to provide mappings from one shading model to another without baking in a consistent manner via procedural graphs. Provide the ability to map procedurals graphs to unlit materials. Reduces runtime memory usage by generating patterns / shading programmatically.
+
+3. **Editability and Extensibility**: Extend runtime editability by exposing logic and interfaces for procedural graphs as well as providing a means to create new or extend existing node definitions.
```

Comment:
This is not the goal of this extension - this extension should simply reference already existing operators/functions and provide a means to store them with a glTF asset.